src/components/forms/transaction/MainSplit.tsx

```diff
@@ -45,6 +45,8 @@ export default function MainSplit({
           : prices.getPrice(account.commodity.mnemonic, txCurrency.mnemonic, d);
 
         setExchangeRate(rate);
+      } else {
+        setExchangeRate(Price.create({ valueNum: 1, valueDenom: 1 }));
```

Comment:
Fixes a bug where when changing txCurrency back, it wouldn't update the exchangeRate back to 1 and thus setting wrong amounts in quantity/value field of the main split